crates/bevy_pbr/src/light.rs

```diff
@@ -425,6 +424,16 @@ pub fn update_directional_light_cascades(
                     transform.compute_matrix(),
                 ))
             }
+            (entity, transform, Projection::Orthographic(projection), camera)
+                if camera.is_active =>
+            {
+                Some((
+                    entity,
+                    projection.area.height() / projection.area.width(),
```

Comment:
```suggestion
                    projection.area.width() / projection.area.height(),
```

the aspect_ratio for `PerspectiveProjection` is the other way around

https://github.com/bevyengine/bevy/blob/695d30bd54af2978dc99f214dda34b568348cf86/crates/bevy_render/src/camera/projection.rs#L153

Reply:
Oops, I even typed it correctly at https://github.com/bevyengine/bevy/pull/7796#discussion_r1115992402